internal/handlers/pg/pgdb/pool.go

```diff
@@ -203,11 +205,53 @@ func (pgPool *Pool) Schemas(ctx context.Context) ([]string, error) {
 	return res, nil
 }
 
+// Collections returns a sorted list of FerretDB collection names.
+func (pgPool *Pool) Collections(ctx context.Context, schema string) ([]string, error) {
+	tx, err := pgPool.Begin(ctx)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+	defer tx.Rollback(ctx)
+
+	settings, err := pgPool.getSettingsTable(ctx, tx, schema)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	collectionsDoc := must.NotFail(settings.Get("collections"))
+	collections, ok := collectionsDoc.(*types.Document)
+	if !ok {
+		return nil, lazyerrors.Errorf("invalid settings document: %v", collectionsDoc)
+	}
+
+	return collections.Keys(), nil
+}
+
 // Tables returns a sorted list of FerretDB collection / PostgreSQL table names.
 // Returns empty slice if schema does not exist.
+// All tables with prefix "_ferretdb_" are filtered out.
 func (pgPool *Pool) Tables(ctx context.Context, schema string) ([]string, error) {
 	// TODO query settings table instead: https://github.com/FerretDB/FerretDB/issues/125
 
+	tables, err := pgPool.tables(ctx, schema)
+	if err != nil {
+		return nil, lazyerrors.Error(err)
+	}
+
+	filtered := make([]string, 0, len(tables))
+	for _, table := range tables {
+		if strings.HasPrefix(table, collectionPrefix) {
+			continue
+		}
+
+		filtered = append(filtered, table)
+	}
+
+	return filtered, nil
+}
+
+// tables returns a list of PostgreSQL table names.
```

Comment:
... and it also creates settings table. 
Would you please add this to the function comment. 
And, probably, it's better to rename the function as it creates schema, creates settings table..

Reply:
No, it's not. Please see full function body.